cryptor/cryptor.go

```diff
@@ -116,6 +117,7 @@ type EncryptedData struct {
 	IV        []byte                      `json:",omitempty"`
 	Data      []byte
 	Signature []byte
+	Usages    []string `json:",omitempty"`
```

Comment:
goimports leaves this alone unless I put annotations on the Data and Signature fields as well. I'll just move this up under the Labels field to match conventions elsewhere.